specs/InfoBadge/InfoBadge-spec.md

```diff
@@ -0,0 +1,526 @@
+
+![Hierarchical NavigationView with an InfoBadge](images/navview-example5.png)
+
+For overflow scenarios in a top-mode NavigationView (i.e. when the window size changes and NavigationView items are placed inside of the overflow menu), the overflow button has its own InfoBadge (`NavigationView.OverflowButtonInfoBadge`) that can be assigned and changed based on the InfoBadges that may be present on overflow items:
+
+![Overflowing NavigationView with an InfoBadge](images/navview-example6.png)
+
+To access the list of items present in the overflow menu, use the `NavigationView.OverflowItems` collection. You can hook up to the Changed event for this collection to accurately check when items are being moved into the overflow menu. Once you know which items are in the overflow menu, you can update the `NavigationView.OverflowButtonInfoBadge` to represent InfoBadge information/status on the overflow button itself while the overflow navigation items are hidden. 
+
+## Using an InfoBadge in another control
+Use InfoBadge as you would any other control – simply add the InfoBadge markup where you’d like it to appear. Since InfoBadge inherits from `Control`, it has all the built-in positioning properties such as margin, alignment, padding, and more which you can take advantage of to position your InfoBadge exactly where you want it. Once the InfoBadge is positioned, use the `IsOpen` property to make the InfoBadge appear and disappear based on user actions, program logic, counters, etc. Note that the `IsOpen` property is set to `true` by default.
+
+If you’re placing an InfoBadge inside of another control, such as a Button or a ListViewItem, note that it is not recommended for the InfoBadge to be visible past the footprint of the parent control. So if your InfoBadge is inside of a button, for instance, it should not be visible past the corners of the Button’s overall footprint. 
+
+## Changing, hiding, and switching between types of InfoBadge
+To show or hide an InfoBadge, simply set the `InfoBadge.IsOpen` property to the appropriate value. On showing and hiding, the InfoBadge will animate.
+
+You can change the icon or number displayed in an InfoBadge while it is being shown. Decrementing or incrementing a numeric InfoBadge based on user action can be achieved by setting/manipulating the value of `InfoBadge.Value`. Changing the icon of an InfoBadge can be achieved by setting `InfoBadge.IconSource` to a new `IconSource` object. When changing icons, ensure that the new icon is the same size as the old icon to avoid a jarring visual effect.
```

Comment:
Will changing values show the InfoBadge? I'm wondering if a developer has `Value` bound to a property like 'NewMailMessages`, then they have to do extra work on top of that to Show/Hide the InfoBadge itself if that value is `0` or not?

At least for Numeric mode, does it make sense that if `Value == 0` then `IsOpen` is set to `false` and if it's non-zero it's set to `true`?

Reply:
Could there be a behaviour to handle visibility based on value?  Maybe a visibility converter helper used?

Reply:
@mdtauk yeah, we have some helpers in the Toolkit for stuff like this, but not sure there's much directly built-in the SDK to help with out the developer having to do a bunch of extra work. Figured I'd point this out here as a discussion topic as the different modes for the control have different use-cases.